bats/tests/preferences/verify-paths.bats

```diff
@@ -0,0 +1,90 @@
+# Test case 30
+
+setup() {
+    load '../helpers/load'
+    # Ensure subshells don't inherit a path that includes ~/.rd/bin
+    PATH=$(ruby -e 'print ENV["PATH"].split(":").reject{|x| x[".rd/bin"]}.join(":")')
+    export PATH="$PATH"
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start rancher desktop' {
+    run rdctl start --application.path-management-strategy=rcfiles --kubernetes.enabled=false
+    assert_success
+}
+
+@test 'ensure the app is running' {
+    try --max 5 --delay 5 rdctl shell echo abc
+    assert_success
+}
+
+@test 'ensure path-management is managed' {
+    # Looks like starting doesn't necessarily guarantee the files have the managed path spec
+    run rdctl set --application.path-management-strategy=manual
+    assert_success
+    sleep 5
+    run rdctl set --application.path-management-strategy=rcfiles
+    assert_success
+    sleep 5
+}
+
+# Running `bash -l -c` causes bats to hang
+#@test 'bash managed' {
+#    run bash -l -c "which rdctl"
+#    assert_output --partial '.rd/bin/rdctl'
+#}
+
+@test 'ksh managed' {
+    if [ -f "$HOME/.kshrc" ]; then
+        run ksh -c "which rdctl"
```

Comment:
Please add `assert_success`. Also in all the following tests.

Reply:
done

Reply:
done